AUTHORS

```diff
@@ -0,0 +1,23 @@
+E-mail addresses listed here are not intended for support.
+
+py-esp32-ulp authors
+--------------------
+py-esp32-ulp is written and maintained by Thomas Waldmann and various contributors:
+
+- Thomas Waldmann <tw@waldmann-edv.de>
+- Wilko Nienhaus <wilko.nienhaus@gmail.com>
+
+Assembler
+---------
+py-esp32-ulp is an assembler toolchain for the ESP32 ULP (Ultra Low-Power)
+Co-Processor, written in MicroPython.
+Copyright 2018, Thomas Waldmann <tw@waldmann-edv.de>, licenced under the terms
+of The MIT License (MIT).
+
+Preprocessor
+------------
+py-esp32-ulp includes a preprocessor for handling ``#define`` statements in
+source, and for expanding RTC convenience macros in a way that mirrors what the real
+ESP-IDF macros do.
+Copyright 2021, Wilko Nienhaus <wilko.nienhaus@gmail.com>, licenced under the terms
+of The MIT License (MIT).
```

Comment:
diff unhappy, missing newline at end of file.

Reply:
Hehe :) will fix.